packages/core/src/data-editor/data-editor.tsx

```diff
@@ -1416,14 +1416,14 @@ const DataEditorImpl: React.ForwardRefRenderFunction<DataEditorRef, DataEditorPr
         [columns, getCellsForSelection, maxColumnWidth, mergedTheme, minColumnWidth, onColumnResize, rowMarkerOffset]
     );
 
+    const [scrollDir, setScrollDir] = React.useState<GridMouseEventArgs["scrollEdge"]>();
```

Comment:
Why is this state? Can't we track this with a ref so the `onMouseMove` event (which will trigger a lot if you move the mouse) doesn't cause React redraws by default?

Reply:
Fixed